src/utils/vscode.ts

```diff
@@ -35,18 +28,61 @@ export function getScope(section: string, settingName: string): vscode.Configura
   return scoppe;
 }
 
-export async function updateConfig<T>(
+export async function warnIfMigrate(settings: string[]): Promise<boolean> {
+  for (let index = 0; index < settings.length; index++) {
+    const setting = settings[index];
+    const section = setting.split('.')[0];
+    const settingName = setting.replace(section + '.', '');
```

Comment:
Nitpick: I know this code will likely have a relatively short timespan, but is it worth making this a little bit cleaner? e.g. by making the argument something like `settings: OptionAddress[]` where `OptionAddress` is

```ts
interface ConfigOption {
  section: string;
  name: string;
}
```

Reply:
Let me be lazy! 😛 

Adjusted as suggested